content/03.results.md

```diff
@@ -215,7 +215,10 @@ These findings support previous reports of a more aggressive phenotype in tumors
 We further investigated the mutational signature profiles of the hypermutant (TMB > 10 Mut/Mb; N = 3) and ultra-hypermutant (TMB > 100 Mut/Mb; N = 4) tumors and/or derived cell lines from six patients in the OpenPBTA cohort (**Figure {@fig:Fig4}E**).
 Five of six tumors were diagnosed as HGGs and one was a brain metastasis of a MYCN non-amplified neuroblastoma tumor.	
 Signature 11, which has been suggested to result from exposure to temozolomide, was indeed present in tumors with previous exposure to the drug (**Table {@tbl:Table2}**).
-We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with Lynch syndrome (**Table {@tbl:Table2}**), a genetic predisposition syndrome caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.3390/genes5030497].
+We detected the MMR2 signature in tumors of four patients (PT_0SPKM4S8, PT_3CHB9PK5, PT_JNEV57VK, and PT_VTM2STE3) diagnosed with either Constitutional mismatch repair deficiency (CMMRD) or Lynch Syndrome (**Table {@tbl:Table2}**), genetic predisposition syndromes caused by a variant in a mismatch repair gene such as _PMS2_, _MLH1_, _MSH2_, _MSH6_, or others [@doi:10.1136/jmedgenet-2020-107627].
+Three of these patients harbored pathogenic germline variants in one of the aforementioned genes. 
+While we did not find a _known_ pathogenic variant in the germline of PT_VTM2STE3, this patient had a self-reported _PMS2_ variant noted in their pathology report and we did find 19 intronic variants of unknown significance (VUS) in _PMS2_.
+This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch Syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
```

Comment:
```suggestion
This is not surprising since an estimated 49% of germline _PMS2_ variants in patients with CMMRD and/or Lynch syndrome are VUS [@doi:10.1136/jmedgenet-2020-107627].
```